extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -171,6 +173,89 @@ class AmpAccordion extends AMP.BaseElement {
     this.currentState_[contentId] = !isSectionClosedAfterClick;
     this.setSessionState_();
   }
+
+  /**
+   * Handles accordion headers clicks to expand/collapse its content.
+   * @param {!Event} event Click event.
+   * @private
+   */
+  clickHandler_(event) {
+    // Need to support clicks on any children of the header.
+    let header;
+    for (let i = 0; i < this.headers_.length; i++) {
+      const curr = this.headers_[i];
+      if (curr.contains(event.target)) {
+        header = curr;
+        break;
+      }
+    }
+    if (header) {
+      header = dev().assertElement(header);
+      event.preventDefault();
+      this.onHeaderPicked_(header);
+    }
+  }
+
+  /**
+   * Handles accordion key presses ot expand/collapse its content.
+   * @param {!Event} event keydown event.
+   */
+  keyDownHandler_(event) {
+    if (event.defaultPrevented) {
+      return;
+    }
+    const keyCode = event.keyCode;
+    switch (keyCode) {
+      case KeyCodes.UP_ARROW: /* fallthrough */
+      case KeyCodes.DOWN_ARROW:
+        this.navigationKeyDownHandler_(event);
```

Comment:
I recommend removing it. I think it complicates the component for no big win. If we keep it, we have to install the `keydown` event handler on the element instead of the `header` and JS runs on every keyboard hit in the content of an accordion which could be many events.